### PR TITLE
Change App Icon

### PR DESCRIPTION
**Pull request by hutomosaktikartiko**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -13,29 +12,25 @@ dependencies:
...
   cupertino_icons: ^0.1.3
   curved_navigation_bar: ^0.3.3
-  url_launcher:
-  shared_preferences:
-  http:
-  dynamic_theme:
+  url_launcher: null
+  shared_preferences: null
+  http: null
+  dynamic_theme: null
```
Comment (SAMYAK99) [inline pubspec.yaml:20]: Why you have used null here  ?

Comment (hutomosaktikartiko) [reply pubspec.yaml:20]: sorry, maybe you can replace it like the previous one

Comment (SAMYAK99) [reply pubspec.yaml:20]: Revert this change and make PR again .

**Review by hutomosaktikartiko — COMMENTED** [review]
please check my PR again

**Pull request merged**